src/components/AppBar/MobileMenu.tsx

```diff
@@ -1,48 +1,44 @@
 import React from 'react';
-import { StyledMenu } from './NavBar.style';
+import { Link } from 'react-router-dom';
+import Fade from '@material-ui/core/Fade';
 import MapIcon from 'assets/images/mapIconTwoTone';
 import FAQIcon from 'assets/images/faqIconTwoTone';
 import palette from 'assets/theme/palette';
-import { Link } from 'react-router-dom';
+import { StyledMenu } from './NavBar.style';
 import { trackNavigation } from './utils';
 
-const MobileMenu: React.FC<{ open: boolean; closeMenu: () => void }> = ({
-  open,
-  closeMenu,
-}) => {
+const MobileMenu: React.FC<{
+  open: boolean;
+  closeMenu: () => void;
+  onMouseLeave: (e: React.MouseEvent<{}>) => void;
+}> = ({ open, closeMenu, onMouseLeave }) => {
   const onClick = (label: string) => {
     const isMobile = true;
     closeMenu();
     trackNavigation(label, isMobile);
   };
 
-  const onMouseLeave = (e: React.MouseEvent<{}>) => {
-    // Do not close when the user hovers on the top bar (including the button
-    // to close the menu.
-    if (e.clientY > 64) {
-      closeMenu();
-    }
-  };
-
   return (
-    <StyledMenu open={open} onMouseLeave={onMouseLeave}>
-      <Link to="/" onClick={() => onClick('Map')}>
-        <MapIcon color={palette.secondary.main} />
-        <span>Map</span>
-      </Link>
-      <Link to="/learn" onClick={() => onClick('Learn')}>
-        <FAQIcon color={palette.secondary.main} />
-        <span>Learn</span>
-      </Link>
-      <Link to="/data-api" onClick={() => onClick('Data API')}>
-        <FAQIcon color={palette.secondary.main} />
-        <span>Data API</span>
-      </Link>
-      <Link to="/about" onClick={() => onClick('About')}>
-        <FAQIcon color={palette.secondary.main} />
-        <span>About</span>
-      </Link>
-    </StyledMenu>
+    <Fade in={open}>
```

Comment:
I think that ideally we should use the theme context instead of importing the palette directly, that should pick the colors from the `ThemeProvider` wrapping the component, making it truly customizable by composition

```tsx
const { palette }  = useTheme();
render (
  <MapIcon color={palette.secondary.main} />
);
```

and then we can have the variations by wrapping the entire component on a `ThemeProvider` with a different theme

```tsx
<ThemeProvider theme={darkTheme}>
  <MobileMenu />
</ThemeProvider>
```

Reply:
lmk if my edit doesnt reflect what you were getting at here